captura/views.py

```diff
@@ -186,6 +188,7 @@ def capture_study(request, id_estudio, numero_seccion):
     context['data'] = data
     context['id_estudio'] = id_estudio
     context['seccion'] = seccion
+    context['Estudio'] = Estudio
```

Comment:
Yes, but this is already in a lot of views and I didn't think it was a good idea to turn this branch into solving that issue. I will open an issue for that specific bug.